tests/acl/test_acl_outer_vlan.py

```diff
@@ -527,6 +569,13 @@ def _do_verification(self, ptfadapter, duthost, tbinfo, vlan_setup_info, ip_vers
                                     pkt_type=pkt_type,
                                     tagged_mode=tagged_mode,
                                     stage=stage)
+        if stage == INGRESS:
+            # Wait mac
+            pytest_assert(wait_until(30, 1, 0, check_mac_status, duthost,
+                                     ptfadapter, outer_vlan_id, dst_mac, dst_port[0]), "mac table is not updated")
+        elif stage == EGRESS:
```

Comment:
>elif [](http://example.com/codeflow?start=8&length=4)

If both conditions do not match, test fails?

Reply:
updated.